examples/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.js

```diff
@@ -0,0 +1,18 @@
+// TODO: Update the image collection ID with the public data catalog ID
+var dataset = ee.ImageCollection('projects/openet/ensemble/conus/gridmet/monthly/v2_0')
```

Comment:
please update to OpenET/ENSEMBLE/CONUS/GRIDMET/MONTHLY/v2_0

Reply:
Done in https://github.com/google/earthengine-catalog/pull/305/commits/e0ddade76770e2a96f2efd4ae0e6304d64a92b7d